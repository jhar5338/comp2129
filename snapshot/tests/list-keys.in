list keys
set a
list keys
set b
set c
set 1
set 2
set 3
del 3
list keys
bye
