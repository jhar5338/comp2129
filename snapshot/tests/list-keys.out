> no keys

> ok

> a

> ok

> ok

> ok

> ok

> ok

> ok

> 2
1
c
b
a

> bye
